app/docs/nu/yona/app/utils/AppUtils.html

```diff
@@ -204,11 +204,11 @@ <h3>Method Summary</h3>
 </tr>
 <tr id="i9" class="rowColor">
 <td class="colFirst"><code>static void</code></td>
-<td class="colLast"><code><span class="memberNameLink"><a href="../../../../nu/yona/app/utils/AppUtils.html#throwException-java.lang.String-java.lang.Exception-java.lang.Thread-nu.yona.app.listener.DataLoadListener-">throwException</a></span>(java.lang.String&nbsp;className,
+<td class="colLast"><code><span class="memberNameLink"><a href="../../../../nu/yona/app/utils/AppUtils.html#reportException-java.lang.String-java.lang.Exception-java.lang.Thread-nu.yona.app.listener.DataLoadListener-">reportException</a></span>(java.lang.String&nbsp;className,
```

Comment:
Same for this file.

Reply:
These html files are documentation files generated by Android Studio based on the comments added in the .java files for classes and methods. 
These files are not build-time-generated.